grafana/grafana
# Scenes: Render old snapshots

**What is this feature?**
Rendering snapshots created with old dashboards in Scenes.

When creating a Snapshot from Scenes, we transform the data [here](https://github.com/grafana/grafana/blob/main/public/app/features/dashboard-scene/serialization/transformSceneToSaveModel.ts#L278)

The idea of this PR is to do the opposite so Scenes can render a snapshot created from an old dashboard view.

**Which issue(s) does this PR fix?**:



Fixes https://github.com/grafana/grafana/issues/82258

**Special notes for your reviewer:**

**How to test it:** 
* Turn on Scenes FF
* Create a Snapshot from an old dashboard view
* Open the Snapshot in incognito window
* Toggle the view to use Scenes and validate the panels display the same data

Please check that:
